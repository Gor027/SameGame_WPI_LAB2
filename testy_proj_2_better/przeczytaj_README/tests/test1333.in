 ,F, ,FF  ,    F,  , ,  ,,F, ,F ,,, ,,,  ,FFFF,FF, ,,F,,F FF  , 
,F F F  ,F
F,,,,
,,F,,,, F F F  , , ,,F,,  , 
F,FF F FFF , F  , F,FF , , ,F,  F,F, ,F,F,FFF,,F
F F, F, ,F,,,  ,, ,,,, F ,, F  ,FFF,F,,FF, F  F,F F  FFF
