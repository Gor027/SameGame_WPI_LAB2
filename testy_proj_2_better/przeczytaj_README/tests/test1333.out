   ,
 ,FF ,
 F,, F  ,         , ,
,,F, ,F FF,   F,  ,,F , ,F     ,,,  ,,  , F  ,F
F,FF,F,FFFF,F ,, F,FF , ,,,F,  F,F,,,F,F,FFFF,FF    F,,
,,F,,F,F,F,,,F,,,,,,,,F,,,,F,F,FFF,F,,FF,FFF,F,F,F,,FFFFFF,
